tests/multiple-listener-client-test.py

```diff
@@ -0,0 +1,24 @@
+import ucp_py as ucp
+import asyncio
+
+async def tmp():
+    ep1 = ucp.get_endpoint(b'192.168.40.20', 13337)
```

Comment:
Maybe you could use `addr = ucp.get_address().encode('utf-8')` and substitue `addr` for the hardcoded ib addr ?

Reply:
+1